backend/app/utils/config_definitions/queries.py

```
@@ -8,6 +8,7 @@
 
 from app.utils.settings.config import settings
 import json
+from fastapi import Request
```

Ruff (0.8.0)

`fastapi.Request` imported but unused

Remove unused import: `fastapi.Request`

(F401)